rfcs/text/0001-rfc-process.md

```diff
@@ -70,46 +106,124 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs.
+Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will 
+usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in
```

Comment:
Informational RFCs do not need to be speculative. Sometimes, they simply document something. At the IETF - https://tools.ietf.org/html/rfc8216 is an example of an informational RFC.

Reply:
Thank you for the thorough background on informational RFCs. How about I re-word it to (and I'm stealing much from the IETF definition here):

"Informational RFCs are more of a compilation of results from performing a case study or research on a topic related to the DCP. There need not be consensus on the RFC nor any actionable outcome unlike non-informational RFCs."